src/components/GeneralInfo/SystemCard/SystemCard.js

```diff
@@ -159,6 +161,18 @@ class SystemCardCore extends Component {
                   },
                 ]
               : []),
+            ...(hasWorkspace
+              ? [
+                  {
+                    title: 'Workspace',
+                    value:
+                      entity.groups?.length > 0 && entity.groups?.[0]?.name,
+                    size: 'md',
+                    customClass: 'sentry-mask data-hj-suppress',
+                    fullTarget: `/insights/inventory/workspaces/${entity.groups?.[0]?.id}`,
```

Comment:
We should not have this (`/insights/inventory/` specifically) hardcoded in here. The other paths for the links are constructed with help from the pathname, would this not work for this section?

Reply:
in this case most of the location would be discarded in order to arrive at the correct path, so i assumed this would be the better solution

Reply:
> in this case most of the location would be discarded in order to arrive at the correct path
I'm not sure I understand what you mean?

Reply:
Please look into how we can get rid of the `/insights/inventory/`. We are already within `/insights/inventory/` and should not need it. If the path changes in the future we have to remember to change it here as well, so we should try to not have it here in the first place.

Reply:
@xmicha82 we can use the `InsightsLink` component. See how other apps use it: https://github.com/search?q=org%3ARedHatInsights%20%3CInsightsLink&type=code